README.md

```diff
@@ -451,7 +451,7 @@ Terms and conditions for use, reproduction and distribution are under the [Apach
 | [Shadow_Runner](https://github.com/kunjgit/GameZone/tree/main/Games/Shadow_Runner) | 
 | [Underwater_Shoot](https://github.com/kunjgit/GameZone/tree/main/Games/Underwater_Shoot) | 
 | [Gravity_Drops](https://github.com/kunjgit/GameZone/tree/main/Games/Gravity_Drops) | 
-
+| [Sky_Lift_Dash](https://github.com/SanskarSinghiit/GameZone/tree/main/Games/Sky_Lift_Dash) |
```

Comment:
You need to change your repo name with kunjgit as others did

Reply:
ok, mam